core/src/main/java/org/dst/core/table/Index.java

```diff
@@ -0,0 +1,25 @@
+package org.dst.core.table;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Index {
```

Comment:
What does this class do? Could you please add some comments to explain about them?